bindings/http/http.go

```diff
@@ -53,9 +68,17 @@ func NewHTTP(logger logger.Logger) bindings.OutputBinding {
 
 // Init performs metadata parsing.
 func (h *HTTPSource) Init(metadata bindings.Metadata) error {
-	if err := mapstructure.Decode(metadata.Properties, &h.metadata); err != nil {
+	var err error
+	if err = mapstructure.Decode(metadata.Properties, &h.metadata); err != nil {
 		return err
 	}
+	var tlsConfig *tls.Config
+	if h.metadata.MTLSEnable == "true" {
```

Comment:
Do we need this option? Or can we just infer it if mtlsClientCert, mtlsClientKey, mtlsRootCA are set?

*If* we do need this option, please use `utils.IsTruthy` to check for truthy values

Reply:
removed, can be inferred from provided certs